### PR TITLE
My awesome pull request!!!

### PR DESCRIPTION
**Pull request by ghost**
Look at this great thing!

### CODE REVIEW

File: skorzeli-Buffalo.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# BUFFALO ROCKS!
+
+But if you are in Dearborn, go to Millers!
```
Comment (githubteacher) [inline skorzeli-Buffalo.md:3]: What should I get at Millers @skorzeli ?

**Comment by notsig11** [comment]
Looks fine to me

File: skorzeli-Buffalo.md (hunk 1 above)
Comment (ghost) [reply skorzeli-Buffalo.md:3]: The only thing you can, a burger and fries, they are very good

Comment (revanisle) [inline skorzeli-Buffalo.md:3]: The title of this section is confusingly misleading, given the body content, and may lead to unnecessary travel expenses and time wasted by people seeking Miller's in Buffalo

**Pull request merged**